### PR TITLE
 [ru] basics sigils - `1.0.3` -> `1.1.0`

### PR DESCRIPTION
**Pull request by IgorPolyakov**

### CODE REVIEW

File: lessons/ru/basics/sigils.md [hunk 1]
```diff
@@ -172,17 +189,38 @@ iex> NaiveDateTime.from_iso8601("2015-01-23 23:50:07") == {:ok, ~N[2015-01-23 23
...
-Мы используем `~u`, потому что она не занята в пространстве строковых меток.
-`_u` означает, что мы хотим использовать символ `u` после тильды.
+Для начала объявим модуль с именем `MySigils` и внутри него создадим функцию `sigil_p`.
+Мы используем `~p`, потому что она не занята в пространстве строковых меток.
+`_p` означает, что мы хотим использовать символ `p` после тильды.
 Функция должна принимать два аргумента: вводимое значение и список.
+
+### Многосимвольные строковые метки
+
+В Elixir 1.15 и выше, идентификаторы строковых меток могут быть последовательностью прописных символов. Это может быть полезно для уточнения назначения строковой метки, предоставляя больше контекста, чем один символ.
```
Comment (nscyclone) [inline lessons/ru/basics/sigils.md:209]:
```suggestion
В Elixir 1.15 и выше идентификаторы строковых меток могут быть последовательностью прописных символов. Это может быть полезно, чтобы уточнить назначение строковой метки, предоставляя больше контекста, чем один символ.
```

File: lessons/ru/basics/sigils.md [hunk 2]
```diff
@@ -172,17 +189,38 @@ iex> NaiveDateTime.from_iso8601("2015-01-23 23:50:07") == {:ok, ~N[2015-01-23 23
...
+...> end
+
+iex> import MySigils
+MySigils
+
+iex> ~REV<foobar>
+"raboof"
+```
+
+Обратите внимание, что для многосимвольных сигилов все символы должны быть прописными. Сигильные функции, такие как sigil_rev или sigil_Rev, вызовут SyntaxError при выполнении.
```
Comment (nscyclone) [inline lessons/ru/basics/sigils.md:226]:
```suggestion
Обратите внимание, что для многосимвольных строковых меток все символы должны быть прописными. Функции строковых меток, такие как `sigil_rev` или `sigil_Rev`, вызовут `SyntaxError` при выполнении.
```

So far sigils have been called "строковые метки", I'd suggest using this term here for consistency.

**Review by nscyclone — CHANGES_REQUESTED** [review]
Thanks @IgorPolyakov 
I've added a couple of minor suggestions to your changes, please take a peek.

**Comment by IgorPolyakov** [comment]
Thanks for the review @nscyclone , I have made the suggested corrections)

**Comment by IgorPolyakov** [comment]
Thanks for the review @nscyclone, I have made the suggested corrections to enhance readability.

**Review by nscyclone — APPROVED** [review]
Thanks @IgorPolyakov

**Pull request merged**